code-reviews/README.md

```diff
@@ -2,51 +2,48 @@
     * AzDO: [Add columns on task board](https://docs.microsoft.com/en-us/azure/devops/boards/sprints/customize-taskboard?view=azure-devops#add-columns)
 2. In the daily standup meeting check tasks pending for review and make sure they have reviewers assigned.
 3. Junior teams and teams new to the process can consider creating separate tasks for reviews together with the tasks themselves.
 
-### Reviewee's guidance
-
-1. When starting to work on a task, break your code into the smallest logically separable units. [TODO: More on size here]
-2. Before submitting a pull request (PR) :
-   * Write tests that cover your changes.
-   * Update the documentation related to your change.
-   * All new and existing tests pass.
-   * The code your wrote follows the style of the project.
-   * Run linting checks.
-   * Do manual testing (run the code) and make sure to not introduce a breaking change.
-
-3. Once done with a unit of code submit a pull request (PR) that follows [pull-request-template.md](pull-request-template.md)
-4. Add one or two reviewers (depending on your project's guidelines). Ideally, you would add at least someone who has expertise and is familiar with the project or the language used and someone who is less familiar with the project or language.
-5. Be open to receive feedback, discuss design/code logic and address all comments as follows:
-   * Resolve the comment if you will make the requested change
-   * Mark the comment as won't fix if you are not going to make the requested changes and explain clearly the reasoning
-   * Ask clarification questions on the review itself (not in a private chat) if you don't understand a reviewer's comment
-   * If a comments thread went beyong 5 or 6 replies and there has been no conclusion, then make direct communication with you reviewer (call them or knock on door)
-   * Mark the comment as pending and clearly state that you are going to address it in a future PR.
-6. If the reviewer's haven't looked at your code in 24-48 hours, ping them to ensure a timely review process.
-7. Once all comments have been addressed and your reviewers have approved you changes, you can submit your code.
+As an author of a PR, you should:
 
-## Reviewer's guidance
+1. Add one or more reviewers (depending on your project's guidelines) to the PR. Ideally, you would add at least someone who has expertise and is familiar with the project or the language used. Adding someone less familiar with the project or the language can aid in verifying the changes are understandable, easy to read, and increases the expertise within the team.
+1. Be open to receive feedback, discuss design/code logic and address all comments as follows:
+   * Resolve a comment, if the requested change has been made.
+   * Mark the comment as "won't fix", if you are not going to make the requested changes and provide a clear reasoning - "I'll do it later" is not an acceptible reason!
```

Comment:
You are refactoring all the content I recently added. And this line here actually says the opposite of what was there. You can occasionaly mark a comment as pending (or won't fix for this specific PR) and you can totally do it later in a following PR. I don't see why this is not an acceptible reason.

Reply:
We can also suggest that when they mark something as won't fix to create a task that it will track it in the following PR.

Reply:
I would like to keep the change.  I prefer this wording, which I also think was the original wording in the document. 
I think avoiding postponing fixes is good advice as later often turns into never.

Reply:
Premise: The PR reflects a task that has well defined scope and acceptance criteria, which the reviewer is well aware of.

Then: If both the author and the reviewer agree a change is required, the task cannot be completed until the change is made.

If the change is out of scope, that's a clear reason: "won't fix; out of scope". Then it's fine creating a new task and addressing the change in another PR.

That said, there are exceptions to every rule, but covering all the corner cases in this document just makes it more confusing for the reader.